### PR TITLE
Add WallstreetBets

### PR DESCRIPTION
**Pull request by abiphull**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,6 +227,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Poloniex](https://poloniex.com/support/api/) | US based digital asset exchange | `apiKey` | Yes | Unknown |
 | [Technical Analysis](https://technical-analysis-api.com) | Cryptocurrency prices and technical analysis | `apiKey` | Yes | No |
 | [VALR](https://docs.valr.com/) | Cryptocurrency Exchange based in South Africa | `apiKey` | Yes | Unknown |
+| [WallstreetBets Api](https://dashboard.nbshare.io/apps/reddit/api/ | WallstreetBets Stock Comments Sentiment Analysis | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:230]:
```suggestion
| [WallstreetBets](https://dashboard.nbshare.io/apps/reddit/api/) | WallstreetBets Stock Comments Sentiment Analysis | No | Yes | Unknown |
```

Remove `API` of title and close link with `)` character, please.

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @abiphull 😃 

Can you resolve the review, please?

File: README.md (hunk 1 above)
Comment (abiphull) [reply README.md]: my bad, thanks for correcting it. I have accepted the suggestions.

**Review by matheusfelipeog — APPROVED** [review]
A big THANKS for your update @abiphull

**Pull request merged**